src/subapps/admin/components/Views/ViewStatisticsProgress.tsx

```diff
@@ -15,7 +15,7 @@ export const ViewStatisticsProgress: React.FunctionComponent<ViewStatisticsProgr
   const label =
     percent === 100
       ? `last indexed: ${moment(props.lastIndexed)
-          .startOf()
+          .startOf('days')
```

Comment:
for some reason, this gives an error with 'days' in local. Not related to subapps.